usecases/cluster/delegate.go

```diff
@@ -241,6 +249,25 @@ func (d *delegate) sortCandidates(names []string) []string {
 	return names
 }
 
+// updater a function which updates node information periodically
+func (d *delegate) updater(period, minPeriod time.Duration, du func(path string) (DiskUsage, error)) {
+	t := time.NewTicker(period)
+	defer t.Stop()
```

Comment:
will this ever be called? looks like loop will never exit

Reply:
This function is called in the init() and remains alive throughout the entire program.